### ISSUE TITLE
Can't save ERD diagram

### ISSUE THREAD

**Issue by beccon4** [dbeaver/dbeaver#20668]
### Description

Saving an ERD diagram based on a MySQL database results in the error message "Cant't determine diagram file". It is possible though to save the diagram into a new file.

### DBeaver Version

23.1.2

### Operating System

Ubuntu 22.04

### Database and driver

MySQL

### Steps to reproduce

Create a ERD diagram based on some tables. Add some virtual foreign key constraints. Then try to save the file.

**Comment by E1izabeth** [comment]
I can't reproduce your issue. Could you describe in more details how do you create this diagram. Exact steps to reproduce or a video would be really helpful.

**Comment by E1izabeth** [comment]
It's been a while since no update here. If the issue is still actual please let me know and provide additional information.

**Comment by carlospauluk** [comment]
This problem is still happening here to me, with DBeaver 23.2.3.
I just create an ERD diagram with some tables from my Oracle Schema.
I save the diagram, and after I close DBeaver and then reopen it, and reopen the Diagram, when I try to save it shows:

![image](https://github.com/dbeaver/dbeaver/assets/949338/f2e3627c-06a8-4b93-82ff-5ef3d50394d7)

**Comment by E1izabeth** [comment]
Could you clarify how exactly did you create the diagram?
This way or some other?
![image](https://github.com/dbeaver/dbeaver/assets/28875055/00ec0ecb-c3b0-4ba7-bae1-f9162b808bba)

**Comment by carlospauluk** [comment]
Through here...

![image](https://github.com/dbeaver/dbeaver/assets/949338/0f923b5b-da4a-4476-b8ac-441c967770b7)

Then I just "Save As" in another directory. After that, when I open that file and make any changes to it, and try to save (i.e.: CTRL + S), it gives me the "Can't determine diagram file")

**Comment by E1izabeth** [comment]
Thank you for details

**Comment by E1izabeth** [comment]
I was managed to reproduce it.

**Comment by davidmachadoSP** [comment]
Same here... Version 24.0.5.202405191503

I went to Menu->Database->Create New ER Diagram, then I followed the wizard, choose tables and it was generated.

After some modifications I saved the model. After a while, I changed the path using SAVE AS menu, then I choose another path and saved it (it was on onedrive desktop). Then it shows me some message about re-open on ER associated file. Then it closes the DBeaver and reaponed with the new path. Then and on the error message starts to been showed.

Always I click SAVE button it shows:

Can't determine diagram file

Then I need to close the message, click on SAVE AS, select the path and the same file.

After that it shows me a message if I want to re-write the file, I click on YES, and the second message to re-open when clicked YES starts the problem again...
